components/dashboard/src/menu/OrganizationSelector.tsx

```diff
@@ -27,9 +27,10 @@ export default function OrganizationSelector() {
     const getOrgURL = useGetOrgURL();
     const configurationsAndPrebuilds = useHasConfigurationsAndPrebuildsEnabled();
     const showPrebuildMenuItem = useFeatureFlag("showPrebuildsMenuItem");
+    const isDedicated = useFeatureFlag("enableDedicatedOnboardingFlow");
```

Comment:
Base on current [FF settings](https://app.configcat.com/08da1258-64fb-4a8e-8a1e-51de773884f6/08da1258-6541-4fc7-8b61-c8b47f82f3a0/08db2c32-4cf8-4951-8835-189864097ded?settingId=83579), we could depend on this FF

Reply:
Uh, isn't this a dead feature flag we are going to remove? :confused: 
I would have expected we have another signal that let's us tell apart this case, e.g. block the "admin" user from these kind of operations. Wouldn't that be more stable than this? This feels a tad too brittle for my taste. :thinking: